src/herbs2rest.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
```

Comment:
recommended:
```js
module.exports = {
  generateRoutes: require('./generateRoutes'),
  generateControllers: require('./generateControllers') // former: generateRoutesWithHerbarium
}
```

this implies not only changing this file but also changing the `generateRoutesWithHerbarium` exports (including its own name)